Main.java

```diff
@@ -9,7 +11,53 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // creates a scanner for user input
+    Scanner input = new Scanner(System.in); 
+
+    //prompt user for the size of pizza
+    System.out.println("What is the size of pizza (in cm)");
+    //intialize the order 
+    double customerOrder = input.nextDouble();  
+
+    
+    //check to see how much it will cost 
+    if(customerOrder <= 20){ 
+      //let them know its gonna be a lil cute on
+      System.out.println("we are going to make you a cute little pizza!"); 
+      double Price1 = customerOrder * 0.50; 
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. This means it should start with a lowercase letter.